src/applications/financial-status-report/containers/IntroductionPage.jsx

```diff
@@ -54,139 +54,134 @@ const IntroductionPage = ({ route, formId }) => {
-            <ul>
-              <li>
-                <strong>Additional assets.</strong>
-                This includes cash in a checking or savings account, stocks and
-                bonds, real estate, cars, jewelry, and other items of value.
-              </li>
-              <li>
-                <strong>Monthly living expenses.</strong>
-                These include housing, food, and utilities (like gas,
-                electricity, and water).
-              </li>
-              <li>
-                <strong>Installment contracts or other debts.</strong>
-                These include car loans, student loans, credit card debt, and
-                other debts or purchase payment plans.
-              </li>
-              <li>
-                <strong>Other living expenses.</strong>
-                These include expenses like clothing, transportation, child
-                care, or health care.
-              </li>
-              <li>
-                <strong>If you’ve ever declared bankruptcy,</strong>
-                you’ll need any related documents.
-              </li>
-            </ul>
-            <p>
-              If you need help with your request,{' '}
-              <a
-                href="https://www.va.gov/vso/"
-                className="vads-u-margin-left--0p5"
-              >
-                contact a local Veterans Service Organization (VSO).
-              </a>
-            </p>
-            <va-alert status="info" uswds visible background-only>
-              <p className="vads-u-margin-top--0p25">
-                <strong>Why do we ask for this information?</strong>
-              </p>
-              <p>
-                We want to make sure we fully understand your financial
-                situation. If you’re married, we also need to understand your
-                spouse’s financial situation. This helps us make the best
-                decision on your request. We will not take collections on your
-                household assets.
-              </p>
-              <p className="vads-u-margin-bottom--0">
-                <strong>Note:</strong> We keep this information confidential. We
-                won’t use or share this information to collect on your income or
-                any assets.
-              </p>
-            </va-alert>
-          </li>
-          <li className="process-step list-two">
-            <h3 className="vads-u-font-size--h4">Submit your request</h3>
-            <p>
-              We’ll take you through each step of the process. It should take
-              about 60 minutes.
+      <va-process-list
+        uswds
+        class="vads-u-margin-left--neg2 vads-u-padding-bottom--0"
+      >
+        <va-process-list-item header="Prepare">
+          <p>
+            You’ll need this information for you (and your spouse if you’re
+            married):
+          </p>
+          <ul>
+            <li>
+              <strong>Work history for the past 2 years.</strong>
```

Comment:
This was probably existing, but since we're in here now: I noticed there's no trailing space on any of the strong tags in this section so they kind of bleed together with the following text. You can try adding a space within the `strong` tag or adding on `{' '}` after the tag to force a space in if the linter isn't happy.

Reply:
Great catch, I'll fix that!